### PR TITLE
Update content to reflect the changes to the Support for early career teacher service

### PR DESCRIPTION
**Pull request by Lloyd-K**
### Context

Content has been updated to reflect the changes to the Support for early career teacher service:

* no need to sign in
* no progress tracking

- Ticket: 

https://dfedigital.atlassian.net/browse/CST-2111?atlOrigin=eyJpIjoiZTdjYTI0ZTkxMDg0NGJlNmIzYzdmZjMxMWNkZjcwN2MiLCJwIjoiaiJ9

### CODE REVIEW

File: app/views/schools/core_programme/materials/show.html.erb [hunk 1]
```diff
@@ -7,12 +7,10 @@
...
 		<h1 class="govuk-heading-l">Changing your accredited materials</h1>
 		
-		<p class="govuk-body">Each set of accredited materials is different. Before you change your materials:</p>
-		<ul class='govuk-list govuk-list--bullet'>
-  			<li>make sure the materials you want to change to are right for your school</li>
-  			<li>if your early career teachers and mentors have started their programme, they will lose any progress on the service</li>
-		</ul>
-		<p class="govuk-body">If you want to change the materials you are using, contact: <%= render MailToSupportComponent.new %></p>
+		<p class="govuk-body">Each set of accredited materials is different. It's important to make sure 
+   the materials you choose are right for your school.</p>
```
Comment (leoapost) [inline app/views/schools/core_programme/materials/show.html.erb:11]: It's fine to have this paragraph in a single line in the source code

```suggestion
		<p class="govuk-body">Each set of accredited materials is different. It's important to make sure the materials you choose are right for your school.</p>
```

**Review by leoapost — APPROVED** [review]
It looks ok. I've made a small suggestion in the comments, but up to you really.

File: app/views/schools/core_programme/materials/show.html.erb (hunk 1 above)
Comment (frankieroberto) [inline app/views/schools/core_programme/materials/show.html.erb]: Typographic apostrophe
```suggestion
		<p class="govuk-body">Each set of accredited materials is different. It’s important to make sure the materials you choose are right for your school.</p>
```

File: app/views/schools/core_programme/materials/show.html.erb [hunk 2]
```diff
@@ -7,12 +7,9 @@
 		<span class="govuk-caption-l"><%= @school.name %></span>
 		<h1 class="govuk-heading-l">Changing your accredited materials</h1>
 		
-		<p class="govuk-body">Each set of accredited materials is different. Before you change your materials:</p>
-		<ul class='govuk-list govuk-list--bullet'>
-  			<li>make sure the materials you want to change to are right for your school</li>
-  			<li>if your early career teachers and mentors have started their programme, they will lose any progress on the service</li>
-		</ul>
-		<p class="govuk-body">If you want to change the materials you are using, contact: <%= render MailToSupportComponent.new %></p>
+		<p class="govuk-body">Each set of accredited materials is different. It's important to make sure the materials you choose are right for your school.</p>
```
Comment (frankieroberto) [inline app/views/schools/core_programme/materials/show.html.erb:10]: Curly apostrophe.

```suggestion
		<p class="govuk-body">Each set of accredited materials is different. It’s important to make sure the materials you choose are right for your school.</p>
```

**Review by frankieroberto — APPROVED** [review]

**Pull request merged**